res/controllers/Hercules DJ Control MP3 e2.bulk.xml

```diff
@@ -1,13 +1,12 @@
 <?xml version='1.0' encoding='utf-8'?>
-<MixxxControllerPreset mixxxVersion="1.11+" schemaVersion="1">
+<MixxxControllerPreset mixxxVersion="2.0+" schemaVersion="1">
     <info>
         <name>Hercules DJ Control MP3 e2 / MP3 LE / Glow</name>
-        <author>Neale Pickett</author>
+        <author>Yoan Ancelly</author>
         <description>Hercules DJ Control MP3 e2 / MP3 LE / Glow mapping. These three controllers have identical controls and share the same mapping.</description>
         <manual>hercules_djcontrol_mp3_e2</manual>
         <devices>
             <product protocol="bulk" vendor_id="0x06f8" product_id="0x0b105" in_epaddr="0x82" out_epaddr="0x03" />
-            <product protocol="bulk" vendor_id="0x06f8" product_id="0x0b120" in_epaddr="0x82" out_epaddr="0x03" />
```

Comment:
Why was this removed? Will it still work with the other controllers?

Reply:
@0xYoan Ping

Reply:
Please dont remove this line, it is needed for the Hercules Glow (Green Version).

Here the dmesg from the Hercules Glow Green-Version:

[10496.108493] usb 2-2.1: new full-speed USB device number 8 using ehci-pci
[10496.198155] usb 2-2.1: New USB device found, idVendor=06f8, idProduct=b120, bcdDevice= 1.11
[10496.198165] usb 2-2.1: New USB device strings: Mfr=1, Product=2, SerialNumber=0
[10496.198171] usb 2-2.1: Product: DJControl MP3 LE
[10496.198174] usb 2-2.1: Manufacturer: Hercules